content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md

```diff
@@ -79,35 +79,20 @@ The project containing the sample CorDapp opens.
 
 `corda-cli network config docker-compose solar-system`
 
-3. Deploy your network by building the network deployment Docker file using Corda CLI:
-  `corda-cli network deploy -n solar-system -f solar-system.yaml > solar-system-compose`
-
-    The `-n` here is the name of the network. The `-f` is the network definition file.
-
-    After you run the command, you will see a new `docker-compose.yaml` file in your project. This contains all of the details for your network, which is now ready to go.
-
-4. Run a Gradle command to build the CorDapp and generate the `.cpk` files:
+3. Run a Gradle command to build the CorDapp and generate the `.cpk` files:
 
   `gradlew build`
 
-5. Use the `cordapp-builder` CLI utility to bundle the `.cpk` files into an easy-to-use `.cpb` file:
-
-  `cordapp-builder create --cpk contracts\build\libs\corda5-solar-system-contracts-cordapp.cpk --cpk workflows\build\libs\corda5-solar-system-workflows-cordapp.cpk -o corda5-hello-solarsystem.cpb`
-
-This command specifies the `.cpk` files of your CorDapp. Most CorDapps will have two files: a `contracts` file and a `workflows` file.
-
-This command also specifies the output file. In this case, the output file is `corda5-hello-solarsystem.cpb`.
-
-  If the command is successful, there is no output.
+4. Run the `gradle cpk` [task in IntelliJ](https://www.jetbrains.com/help/idea/work-with-gradle-tasks.html#gradle_tasks) to build the CorDapp package (`.cpk`) files of your CorDapp. Most CorDapps will have two files: a `contracts` file and a `workflows` file.
 
-  If you have an error in your command, the segment with the error is returned in red.
+5. Run the `gradle cpb` task in IntelliJ to build the CorDapp package bundle (`.cpb`).
```

Comment:
The gradle `build` task actually also covers the cpk and cpb, so these 2 steps aren't necessary, although some of the conent/links here maybe needs to be incorperated in step 3.